docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -0,0 +1,287 @@
+
+Subscription Foundation uses the [Channels toolkit](/api-docs/channels/guide/overview#channels-toolkit) to display the custom subscription channel in the **Channel Manager** of a store's control panel alongside other sales channels. 
+
+## Software requirements 
+* [Node.js](https://nodejs.org/en/) 14.17.0
+* The [npm](https://www.npmjs.com/) package manager
+* A [supported SQL database engine](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema/data-sources/), either SQLite or another of your choice
+
+## Configure accounts
+
+<!-- theme: info -->
+> #### To configure your accounts you need to make sure the following requirements are met:
+> * Because there is a [store email address constraint](/api-docs/apps/guide/developer-portal#store-email-address-constraint?source=subscription-foundation) on draft and private apps, you need to make sure that both your sandbox store and your Dev Portal account use the same email address. 
+> * The sandbox store must support multi-storefront sales. For information on configuring multi-storefront, see [Multi-Storefront](https://support.bigcommerce.com/s/article/Multi-Storefront?language=en_US).
+
+To configure your accounts complete the following steps:
+1. To develop and test apps, you need a BigCommerce sandbox store. If you don't have one, [Create a Sandbox Store](/api-docs/partner/getting-started/create-a-sandbox-store?source=subscription-foundation).
+2. To register apps and create app API accounts, you need a BigCommerce Dev Portal account. If you don't have one, create a [Dev Portal account](https://devtools.bigcommerce.com/?source=subscription-foundation).
+
+
+
+## Fork and install the source repository
+To fork the repository complete the following steps:
+
+1. Fork the [subscription-foundation repository (GitHub)](https://github.com/bigcommerce/subscription-foundation) to your GitHub account.
+2. Clone your fork to your local development environment.
+3. Navigate to your root directory of your cloned repository and install the default packages for your app by running the following command:
+
+```shell title="Install packages"
+npm install
+```
+## Create an HTTPS tunnel
+
+After you successfully install the packages, prepare to expose the app to the internet so you can install it on your sandbox store.
+
+Because you must serve apps over fully-qualified publicly accessible URLs, this guide uses a tool called [ngrok](https://ngrok.com/docs) to open HTTP tunnels that securely expose `localhost` ports to the internet.  
+
+To get started with ngrok, follow the [Create an HTTPS Tunnel](/api-docs/apps/tutorials/sample-app-nextjs/step-2-connect#create-an-https-tunnel) section of the sample app tutorial. If your app does not run on port 3000, replace `3000` with the port of your app server.
+
+<!-- theme: important -->
+> #### Make note of your ngrok ID
+> After you successfully start ngrok, make sure to note your ngrok ID. You will use it later.
```

Comment:
Traci, can you please add a note saying that the ngrok id will change if you restart the service